core/web/loader/job_spec_errors.go

```diff
@@ -0,0 +1,60 @@
+package loader
+
+import (
+	"context"
+
+	"github.com/graph-gophers/dataloader"
+
+	"github.com/smartcontractkit/chainlink/core/services/chainlink"
+	"github.com/smartcontractkit/chainlink/core/services/job"
+	"github.com/smartcontractkit/chainlink/core/utils/stringutils"
+)
+
+type jobSpecErrorsBatcher struct {
+	app chainlink.Application
+}
+
+func (b *jobSpecErrorsBatcher) loadByJobIDs(_ context.Context, keys dataloader.Keys) []*dataloader.Result {
```

Comment:
Please do not throw away the context

Reply:
We are not really doing anything with the context here... should i pass it around to the ORM call and wrap that context as part of the whole process in the ORM?